src/webgl/light.js

```diff
@@ -592,13 +592,60 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
  * @alt
  * light with slider having a slider for varying roughness
  */
-p5.prototype.imageLight = function(img){
+p5.prototype.imageLight = function (img) {
   // activeImageLight property is checked by _setFillUniforms
   // for sending uniforms to the fillshader
   this._renderer.activeImageLight = img;
   this._renderer._enableLighting = true;
 };
 
+/**
+ * The `panorama(img)` method adeptly transforms images such as
```

Comment:
How do you feel about this as an update to the wording for this paragraph? I tried to simplify the language a little, and describe more what the content of the image should be.
```
 * The `panorama(img)` method transforms images containing
 * 360-degree content, such as maps or HDRIs, into immersive
 * 3D backgrounds that surround your scene. This is similar to calling
 * `background(color)`; call `panorama(img)` before drawing your
 * scene to create a 360-degree background from your image. It
 * operates on the concept of sphere mapping, where the image is
 * mapped onto an infinitely large sphere based on the angles of the
 * camera.
```

Reply:
Sounds good!